Source/WebKit/UIProcess/ios/WebPageProxyIOS.mm

```diff
@@ -1151,10 +1150,6 @@ static FloatSize fullscreenPreferencesScreenSize(CGFloat preferredWidth)
 void WebPageProxy::hardwareKeyboardAvailabilityChanged()
 {
     auto hardwareKeyboardState = currentHardwareKeyboardState();
-    if (hardwareKeyboardState == internals().hardwareKeyboardState)
```

Comment:
That's a good point! Since the notification service is blocked in the WebContent process now, this should not be an issue anymore.

Thanks for reviewing!